workflow/rules/neighborhoods.smk

```diff
@@ -16,7 +16,8 @@ rule create_neighborhoods:
 	output: 
 		enhList = os.path.join(RESULTS_DIR, "{biosample}", "Neighborhoods", "EnhancerList.txt"),
 		geneList = os.path.join(RESULTS_DIR, "{biosample}", "Neighborhoods", "GeneList.txt"),
-		neighborhoodDirectory = directory(os.path.join(RESULTS_DIR, "{biosample}", "Neighborhoods"))
+		neighborhoodDirectory = directory(os.path.join(RESULTS_DIR, "{biosample}", "Neighborhoods")),
+		processed_genes_file = os.path.join(RESULTS_DIR, "{biosample}", "processed_genes_file.bed"),
```

Comment:
Maybe do it at the beginning of the pipeline as a separate rule, so it's only created once

Reply:
The snakemake rule would look complicated since we allow custom/alternative gene files. So we'd need handling to create the common one and then the dataset specific ones. 
I did something similar for creating a rule to fit the HiC data to the powerlaw, where we have common powerlaw and dataset specific. Lmk if there's an easy way to do this in Snakemake. Otherwise, I figure the bedtools step should be very fast anyways

Reply:
I see. I guess no point dwelling on it since the file sizes are small